app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,134 @@
+publish API specs from any version -- no matter which runtime group the version
+is in. However, application registration is only supported for Service versions
+running in the default runtime group.
+
+So, if you want a Service version to be available for application registration,
+create the version in the *default* runtime group. This means that you can have a
+mix of Service versions running in different groups for different purposes.
+
+For example, you might have:
+
+* The latest version of a Service published to the Dev Portal with the latest
+spec, running in the *default* group with application registration enabled.
+
+* A previous version of the Service published to the Dev Portal,
+running in the *default* group, but with application registration disabled.
+
+* Another previous version of the Service, published to the Dev Portal, and
+running in a *custom* runtime group. This version would only have the documentation
+available, and developers would not have access to application registration for
+this version.
+
+Out of these three scenarios, only the Service version in the first scenario
+would be available for application registration.
+
+## Multiple runtime groups
+{:.badge .enterprise}
+
+In addition to the default runtime group, you can also configure two additional
+custom groups. Splitting runtime instances and configuration into multiple
+runtime groups provides multiple specialized environments in one account.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+[Set up and manage runtime groups &rarr;](/konnect/configure/runtime-manager/runtime-groups/manage)
+
+## Administration
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions.
+
+[Administer teams &rarr;](/link)
+
+## Global configuration
+
+For each runtime group, you can spin up runtime instances and configure global
+options for the following {{site.base_gateway}} objects:
+* Consumers
+* Plugins
+* Upstreams
+* Certificates
+* SNIs
+
+A **global** object is a set of configurations that apply to, or can be used
+by, all objects in a runtime group. For example, if you set up a Proxy Caching
+plugin in the default runtime group and set it to `Global`,
+the plugin configuration will apply to all Service versions in the group.
+
+When there are multiple runtime groups, configuration for any of the global
+objects only applies to the runtime group that it was created in. Consumers and
+their authentication mechanisms don't carry over to other runtime groups.
```

Comment:
I suppose we can keep this explanation since it is useful and still accurate.
Both the plugins and consumers explanation is helpful information. Anyway it can be rephrased to put less stress on the "global" part?

The consumers paragraph literally speaks to how a consumer is not global but does apply to an entire RG.

Reply:
> The consumers paragraph literally speaks to how a consumer is not global but does apply to an entire RG.

This feels like a terminology mismatch. Here's what we have in the UI:
<img width="708" alt="Screen Shot 2022-03-25 at 3 31 01 PM" src="https://user-images.githubusercontent.com/54370747/160210518-27c88273-6571-4fc1-b556-7de6fe3cf2e8.png">

Either the term needs to change, or the description needs to say "All services, routes, and consumers in the runtime group".

Reply:
Mmm, you're right, and you did clarify in that what global means in the context of the RG already.
I will have to revisit plugins in Konnect, with Ross and team separately. It's going to start deviating from plugins for on-prem.